### ISSUE TITLE
Can't tile to quarter-screen on 1080p monitor

### ISSUE THREAD

**Issue by Fmstrat** [glowing-bear/glowing-bear#1066]
I use quarter-tiling a lot, where many messaging apps take up a quarter of the screen.

With Glowing Bear (built in Electron) once I get close to quarter size it starts to cut off content on the far right-hand side. Any chance we could just keep wrapping content?

**Comment by lorenzhs** [comment]
Can you show a screenshot? I can't reproduce this. I think you're just getting the mobile view as soon as the width goes below 968px (or something like that), where the buffer list slides out from the left. Once you select a buffer, you should be able to see its lines in full.

**Comment by Fmstrat** [comment]
Yes, that was it. For some reason the buffer list wasn't sliding, it was stuck out, but after a restart all seems good.

I did find an unrelated bug where if the "Always show nick list" is checked, and the app starts in mobile mode, and starts in a channel, it doesn't show up unless you leave the channel and come back. Not sure if you want me to file an issue for that.

**Comment by lorenzhs** [comment]
Just to be clear, you meant:
- have a channel open, enable "always show nick list", disconnect
- reconnect, GB re-opens the channel, no nicklist shown?

Because I can't reproduce that (in Firefox, but hopefully that shouldn't matter)

**Comment by Fmstrat** [comment]
Yes, that is the case. Quarter-tile the window (so it's in mobile mode), and that's when it does it. This is in Electron, but yea, it should be the same.

**Comment by AStove** [comment]
I can see what you mean.

Part of the bufferlist is cut off and the buttons.

![image](https://user-images.githubusercontent.com/23409814/71415883-4c78fc00-265e-11ea-86c2-cbad3c5e7fa7.png)
